src/smcntrpmf.adoc

```diff
@@ -16,10 +16,10 @@ Smcntrpmf remedies these issues by introducing privilege mode filtering for the
 
 mcyclecfg and minstretcfg are 64-bit registers that configure privilege mode filtering for the cycle and instret counters, respectively.  
 
-[cols="^1,^1,^1,^1,^1,^1,^5",stripes=even,options="header"]
+[cols="^1,^1,^1,^1,^1,^1,^5,^1",stripes=even,options="header"]
 |====
-|63 |62 |61 |60 |59 |58 |57:0
-|0 |MINH |SINH |UINH |VSINH |VUINH |_WPRI_ 
+|63 |62 |61 |60 |59 |58 |57:0 |56
```

Comment:
I believe "57:0" should now be just "57".

Reply:
Fixed.

Reply:
Ah, I missed this discussion.  These counters count fixed events, so they don't use these cfg registers as event selectors.  57:0 should be WPRI, all those bits are reserved.

Reply:
If Smcntrpmf is going to match Sscofpmf wrt to the top byte of mhpmevent bits (which has been my understanding), then that should include mirroring what is spec'ed for bits [57:56].

Reply:
And it already does that.  m*cfg[57:0] is listed as WPRI, which matches the change we're making to Sscofpmf to make mhpmeventX[57:56] WPRI.

Note that it doesn't exactly match, since Zicntr counters do not support interrupt-based sampling.  So the OF bit (bit 63) is 0 in m*cfg.

Reply:
I knwo you're still catching up with the posts, but the status quo for now is the the integrated Sscofpmf spec will match the original ratified spec in saying nothing about bits [55:0].  See my earlier post about the difference at issue (and that the Priv spec already provides a definition for all these low bits).

Any change (from the ratified spec) to how the lower bits are spec'ed would be a matter for the ARC to discuss and decide.  In the meantime I assume Sscntrpmf would mirror what Sscofpmf spec's?

Reply:
Unlike Sscofpmf which defined the top byte of mhpmevent*, Smcntrpmf defined a new CSRs - mcyclecfg, minstretcfg - and Smcntrpmf made bits 57:0 of that new CSR as WPRI. That should not be changed. So bits 57:0 in mcyclecfg, minstretcfg should remain WPRI.

Reply:
Understood and makes sense.  But just to note, Smcntrpmf is choosing to not mirror what Sscofpmf and the Priv spec says for the low bits of the mhpmevent registers.  But it IS choosing to at least mirror Sscofpmf for bits [63:56].

Reply:
To summarize:
1. The table in Smcntrpmf does not change. The bits 57:0 remain WPRI
2. The statement in Smcntrpmf "Bits 57:56 are reserved for possible future modes." should be replaced by "Bits 57:56 are reserved."

Reply:
@ved-rivos : The original Smcntrpmf spec had 56:57 reserved following the ratified Sscofpmf spec. The decision to make 56:57 as WPRI happened due to the discussion initiated last month.
https://github.com/riscv/riscv-isa-manual/pull/1582

Since we are reverting that decision for Sscofpmf, shouldn't Smcntrpmf follow as well ?

Reply:
As per @bcstrongx comments, no changes are required to Smcntrpmf.